src/debugProtocol/Debugger.ts

```diff
@@ -113,7 +113,7 @@ export class Debugger {
             'runtime-error' |
             'start' |
             'suspend',
-        /* eslint-disable */
```

Comment:
This was what prompted this fix. it was `disable` when should have been `enable`. Changing to `enable` surfaced many lint issues in the remainder of this file.